common-ui/src/main/res/values/widgets.xml

```diff
@@ -526,6 +526,7 @@
         <item name="android:textColor">@color/emptyFavoriteActionTextColor</item>
     </style>
 
-
+    <!-- Notify Me component -->
+    <style name="Widget.DuckDuckGo.NotifyMeView"/>
```

Comment:
this is empty, why do you need it? it can be removed

Reply:
Updated.